lib/plugins/aws/package/compile/events/apiGateway/lib/usagePlan.test.js

```diff
@@ -288,3 +289,100 @@ describe('#compileUsagePlan()', () => {
     });
   });
 });
+
+describe('UsagePlan', () => {
+  const burstLimit = 98;
+  const rateLimit = 99;
+
+  const limit = 101;
+  const offset = 2;
+  const period = 'MONTH';
+
+  const quota = {
+    limit,
+    offset,
+    period,
+  };
+
+  const throttle = {
+    burstLimit,
+    rateLimit,
+  };
+
+  const serverlessConfigurationExtension = {
+    provider: {
+      name: 'aws',
+      usagePlan: {},
+    },
+    functions: {
+      foo: {
+        events: [
+          {
+            http: {
+              integration: 'AWS',
+              request: { template: { 'application/x-www-form-urlencoded': null } },
+            },
+          },
+        ],
+      },
+    },
+  };
+
+  it('Should have values for throttle', () => {
+    serverlessConfigurationExtension.provider.usagePlan = { throttle };
+    return runServerless({
+      fixture: 'apiGateway',
+      configExt: serverlessConfigurationExtension,
+      cliArgs: ['package'],
+    }).then(({ cfTemplate }) => {
+      const cfResources = cfTemplate.Resources;
+
+      expect(cfResources.ApiGatewayUsagePlan.Properties.Throttle).to.not.be.undefined;
```

Comment:
Let's remove this test (this is confirmed by below tests already, and in given form just adds extra noise)